website/docs/cloud-docs/projects/best-practices.mdx

```diff
@@ -0,0 +1,36 @@
+---
+page_title: Best Practices - Projects - HCP Terraform
+description: >-
+  Best practices to structure your configuration and HCP Terraform
+  projects
+---
+
+# Project Best Practices
+
+Projects let you group and scope access to your workspaces. You can group related  workspaces into projects and give teams more permissive access to individual projects rather than granting them permissions to the entire organization. 
+
+Projects offer several advantages to help you further develop your workspace strategy:
+
+- **Increased workspace organization**: You can add related workspaces to projects to simplify and organize a team's workspace view.
+- **Simplified management**: You can create project-level permissions and variable sets that apply to all current and future workspaces in the project. For example, you can create a project variable set containing your cloud provider credentials and all workspaces in the project will have access to them.
+- **Reduced risk with centralized control**: You can scope project permissions to only grant teams administrator access to the projects and workspaces they need.
```

Comment:
```suggestion
- **Reduced risk with centralized control**: You can scope project permissions to only grant teams administrator access to the projects and workspaces they need.
```
```suggestion
- **Reduced risk with centralized control**: You can scope project permissions to only grant a team's administrator access to the projects and workspaces they need.
```
not clear to me what we're actually recommending here. is it team admin to be the only one with elevated privileges on a project, or org admins with cross-project privileges?